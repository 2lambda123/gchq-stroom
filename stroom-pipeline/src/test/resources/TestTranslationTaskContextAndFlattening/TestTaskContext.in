<?xml version="1.0" encoding="UTF-8"?>
<SomeData>
  <SomeEvent>
    <SomeTime>01/01/2009:12:00:01</SomeTime>
    <SomeAction>OPEN</SomeAction>
    <SomeUser>userone</SomeUser>
    <SomeFile>D:\TranslationKit\example\VerySimple\OpenFileEvents.txt</SomeFile>
  </SomeEvent>
</SomeData>


