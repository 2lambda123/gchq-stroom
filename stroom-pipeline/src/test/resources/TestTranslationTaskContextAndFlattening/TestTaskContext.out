<?xml version="1.0" encoding="UTF-8"?><Events xmlns:stroom="stroom" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="event-logging:3" xsi:schemaLocation="event-logging:3 file://event-logging-v3.0.0.xsd" Version="3.0.0"><Event Id="2:1"><EventTime><TimeCreated>2009-01-01T12:00:01.000Z</TimeCreated></EventTime><EventSource><System>Example</System><Environment>Example</Environment><Generator>Very Simple Provider</Generator><Device><IPAddress>192.168.0.4</IPAddress><Location><Country>UK</Country><Site>MyMachine</Site><Building>Main</Building><Floor>1</Floor><Room>1aaa</Room></Location></Device><User><Id>userone</Id></User></EventSource><EventDetail><View><Object><Type>Document</Type><Title>UNKNOWN</Title><Path>D:\TranslationKit\example\VerySimple\OpenFileEvents.txt</Path></Object></View></EventDetail></Event></Events>
